cves/CVE-2013-4310.yml

```diff
@@ -6,33 +6,33 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: Broken Access Control
 
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-4310
```

Comment:
We are looking for a _CWE_ here. You can find them here: http://cwe.mitre.org/.  e.g. CWE-20